<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Serch</class>
 <widget class="QWidget" name="Serch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>输入姓名查询联系人</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>711</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTextEdit" name="t_classmate">
      <property name="styleSheet">
       <string notr="true">font: 25 10pt &quot;微软雅黑 Light&quot;;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="b_classmate">
      <property name="text">
       <string>查询同学</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>711</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTextEdit" name="t_workmate">
      <property name="styleSheet">
       <string notr="true">font: 25 10pt &quot;微软雅黑 Light&quot;;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="b_workmate">
      <property name="text">
       <string>查询同事</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>480</y>
     <width>711</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QTextEdit" name="t_relative">
      <property name="styleSheet">
       <string notr="true">font: 25 10pt &quot;微软雅黑 Light&quot;;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="b_relative">
      <property name="text">
       <string>查询亲戚</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>711</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTextEdit" name="t_friend">
      <property name="styleSheet">
       <string notr="true">font: 25 10pt &quot;微软雅黑&quot;;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="b_friend">
      <property name="text">
       <string>查询朋友</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
